### ISSUE TITLE
False errors shown for @apply 

### ISSUE THREAD

**Issue by mjbvz** [microsoft/typescript-styled-plugin#82]
From https://github.com/mjbvz/vscode-lit-html/issues/45

**Repo**
For the code:

```ts
const a = styled.h1`
    @apply --bla;
`
```

**Bug**
False errors reported

This is likely caused by us using the scss language to get the errors instead of normal css language service

**Comment by OnurGvnc** [comment]
`tsconfig.json`

https://github.com/Microsoft/typescript-styled-plugin#linting

it works as a workaround

```json
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-styled-plugin",
        "validate": false
      }
    ]
  }
}
```

**Comment by naoak** [comment]
@mjbvz 
vscode-css-languageservice seems to have already supported @apply rule since v4.0.3 by [this commit](https://github.com/microsoft/vscode-css-languageservice/pull/194/commits/6a0ec4074769a7a1a92a02f86640704d0af74cd1).

package.json in this project (typescript-style-plugin@0.15.0) points vscode-css-languageservice at ~4.0.2, so there is no need to do anything in theory.

Updating dependencies of typescript-lit-html-plugin and vscode-lit-html will be work. 
https://github.com/microsoft/typescript-lit-html-plugin/blob/master/package.json#L22
https://github.com/mjbvz/vscode-lit-html/blob/master/package.json#L154

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)

After updating to that fork, follow up in the styled-components repo if this is still an issue
